pkg/specgen/namespaces.go

```diff
@@ -263,23 +262,13 @@ func ParseNamespace(ns string) (Namespace, error) {
 // form.
 func ParseCgroupNamespace(ns string) (Namespace, error) {
 	toReturn := Namespace{}
-	// Cgroup is host for v1, private for v2.
-	// We can't trust c/common for this, as it only assumes private.
-	cgroupsv2, err := cgroups.IsCgroup2UnifiedMode()
-	if err != nil {
-		return toReturn, err
-	}
-	if cgroupsv2 {
-		switch ns {
-		case "host":
-			toReturn.NSMode = Host
-		case "private", "":
-			toReturn.NSMode = Private
-		default:
-			return toReturn, fmt.Errorf("unrecognized cgroup namespace mode %s passed", ns)
-		}
-	} else {
+	switch ns {
+	case "host":
 		toReturn.NSMode = Host
+	case "private", "":
+		toReturn.NSMode = Private
+	default:
+		return toReturn, fmt.Errorf("unrecognized cgroup namespace mode %s passed", ns)
 	}
 	return toReturn, nil
```

Comment:
(It would be a bit simpler to `return Namespace{…}` in every case of the switch, without the `toReturn` variable.)

Reply:
done